src/module_2/Module_2_EDA_Ex1.md

```diff
@@ -0,0 +1,3152 @@
+all_ordered_items = [item for sublist in data['ordered_items'] for item in sublist]
+
+# Count occurrences of each product
+product_counts = pd.Series(all_ordered_items).value_counts()
+
+# Get the top 10 best-selling products
+top_10_products = product_counts.head(10)
+
+# Get the informarion of these products
+top_10_details = inventory[inventory['variant_id'].isin(top_10_products.index)]
+top_10_details
+```
+
+
+
+
+<div>
+<style scoped>
+    .dataframe tbody tr th:only-of-type {
+        vertical-align: middle;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+
+    .dataframe thead th {
+        text-align: right;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>variant_id</th>
+      <th>price</th>
+      <th>compare_at_price</th>
+      <th>vendor</th>
+      <th>product_type</th>
+      <th>tags</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>1</th>
+      <td>34370361229444</td>
+      <td>4.99</td>
+      <td>5.50</td>
+      <td>whogivesacrap</td>
+      <td>toilet-roll-kitchen-roll-tissue</td>
+      <td>[b-corp, eco, toilet-rolls]</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>34284951863428</td>
+      <td>3.69</td>
+      <td>3.99</td>
+      <td>plenty</td>
+      <td>toilet-roll-kitchen-roll-tissue</td>
+      <td>[kitchen-roll]</td>
+    </tr>
+    <tr>
+      <th>63</th>
+      <td>34081589887108</td>
+      <td>10.79</td>
+      <td>11.94</td>
+      <td>oatly</td>
+      <td>long-life-milk-substitutes</td>
+      <td>[oat-milk, vegan]</td>
+    </tr>
+    <tr>
+      <th>109</th>
+      <td>34284949766276</td>
+      <td>8.49</td>
+      <td>9.00</td>
+      <td>andrex</td>
+      <td>toilet-roll-kitchen-roll-tissue</td>
+      <td>[toilet-rolls]</td>
+    </tr>
+    <tr>
+      <th>116</th>
+      <td>33826465153156</td>
+      <td>1.89</td>
+      <td>1.99</td>
+      <td>clearspring</td>
+      <td>tins-packaged-foods</td>
+      <td>[gluten-free, meat-alternatives, vegan]</td>
+    </tr>
+    <tr>
+      <th>476</th>
+      <td>34537169715332</td>
+      <td>13.69</td>
+      <td>13.79</td>
+      <td>catsan</td>
+      <td>pet-care</td>
+      <td>[pet-care]</td>
+    </tr>
+    <tr>
+      <th>928</th>
+      <td>34284950356100</td>
+      <td>1.99</td>
+      <td>3.00</td>
+      <td>fairy</td>
+      <td>dishwashing</td>
+      <td>[discontinue, swapped, washing-up-liquid]</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+We can observe that the best-selling products include toilet and kitchen rolls, oat milk, vegan packaged foods, pet-care items, and dishwashing products. However, it is important to note that inventory data for some of these top-selling products is missing. Understanding the reasons for this absence of data is crucial; for instance, these products may have been discontinued. Additionally, investigating what these specific products were could provide further insights.
```

Comment:
Given the greatest shift in orders, this results are bias towards latest months, which is fine. But it is worth noting. Maybe the top 10 changed dramatically with sales too!